app/src/main/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
This file should be placed like this:
app/src/
    flavor1/google-services.json
    flavor2/google-services.json